### PR TITLE
Added root_path to urls in masthead controls nav

### PR DESCRIPTION
**Pull request by little9**
Adding the root_path to the urls fixed the nav bar portion of issue #1141 for me. I'm running Sufia in a subdirectory with Apache/Passenger.

### CODE REVIEW

File: app/views/_controls.html.erb [hunk 1]
```diff
@@ -3,9 +3,9 @@
     <div class="col-xs-12 col-sm-5 col-md-6">
       <nav class="navbar navbar-default" role="navigation">
         <ul class="nav navbar-nav">
-            <li <%= 'class=active' if current_page?(root_path) %>><a href="/" >Home</a></li>
-                <li <%= 'class=active' if current_page?(sufia.about_path) %>><a href="/about/" >About</a></li>
-                <li <%= 'class=active' if action_name == "help" %>><a href="/help/" >Help</a></li>
+            <li <%= 'class=active' if current_page?(root_path) %>><a href="<%= root_path %>" >Home</a></li>
+                <li <%= 'class=active' if current_page?(sufia.about_path) %>><a href="<%= root_path %>/about/" >About</a></li>
```
Comment (mjgiarlo) [inline app/views/_controls.html.erb]: I believe this can be `<%= sufia.about_path %>`

File: app/views/_controls.html.erb [hunk 2]
```diff
@@ -3,9 +3,9 @@
...
+                <li <%= 'class=active' if action_name == "help" %>><a href="<%= root_path %>/help/" >Help</a></li>
```
Comment (mjgiarlo) [inline app/views/_controls.html.erb]: I believe this can be `<%= sufia.static_path('help') %>`

**Comment by mjgiarlo** [comment]
Hi, @little9!

Thanks for the pull request. In order for us to merge this code, we'll need to have signed Contributor License Agreements on file for you as an individual and the University of Miami as a whole.  Because this PR is small, you may or may not wish to deal with this process, in which case someone who's already covered by CLAs can implement this.  If you're likely to continue making improvements to Sufia and Hydra code, you may indeed want to get CLAs signed. Please advise!

You can read more about our CLA processes here:

https://wiki.duraspace.org/display/hydra/Hydra+Project+Intellectual+Property+Licensing+and+Ownership

The process itself is fairly simple -- you just need to print out the Hydra Project iCLA and sign it and then send it to legal@projecthydra.org. Same for the cCLA but that document must be signed with someone who has signatory authority for your Miami. Let me know if y'all have any questions about this process, and I'd be happy to help.

**Comment by mjgiarlo** [comment]
@little9's iCLA has been submitted and the cCLA will be on its way soon.  Hang in there!

**Comment by mjgiarlo** [comment]
How's this going, @little9?

**Comment by mjgiarlo** [comment]
@little9 Would you prefer if I took your patch and submitted it on your behalf while we wait on you cCLA?

**Comment by little9** [comment]
Yeah, that's OK. The process is taking a little bit longer that we had first imagined.

**Comment by mjgiarlo** [comment]
OK, cool, @little9. Thanks for the PR.  Closing. 

See: https://github.com/projecthydra/sufia/pull/1277

**Pull request closed without merging**